extensions/todoist/src/helpers/withTodoistApi.tsx

```diff
@@ -15,6 +16,10 @@ export async function initializeApi() {
     baseURL: "https://api.todoist.com/sync/v9",
     headers: { authorization: `Bearer ${accessToken}` },
   });
+  todoistRestApi = axios.create({
+    baseURL: "https://api.todoist.com/rest/v2",
+    headers: { authorization: `Bearer ${accessToken}` },
+  });
```

Comment:
It'd be better in this case to use Doist's TypeScript SDK: https://github.com/doist/todoist-api-typescript

Reply:
Didn't want to add too many requeriments, specially given this request is rather simpler and given most of the codebase is structured around the existing Task object (so parsing it via existing SDK may require me to add a bridge to existing Task object or refactor a lot)

Reply:
Makes sense yes!